src/generate.ml4

```diff
@@ -1446,28 +1421,37 @@ let mmake_cmp_def k ind masks mind =
 	in
 	Command.do_fixpoint Decl_kinds.Global false defs
 
-let using_sym =
-  let name = {
-    mltac_plugin = "containers_plugin";
-    mltac_tactic = "using_sym";
-  } in
-  let entry = {
-    mltac_name = name;
-    mltac_index = 0;
-  } in
-  let tac args _ =
-    let map v =
-      let id = Tacinterp.Value.cast (topwit wit_ident) v in
-      { Tacexpr.delayed = fun _ sigma -> Sigma.here (Constrintern.global_reference id) sigma }
-    in
-    let args = List.map map args in
-    Auto.h_auto None args (Some [])
+let using_sym_tac = ref (None:Tacexpr.ml_tactic_entry option)
```

Comment:
I am using this option reference to ensure `using_sym` is not multiply defined. Is there a more idiomatic way to accomplish this?